ibm/data_source_ibm_list_scopes_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
```

Comment:
File names should match the resource name as well. It should be
data_source_ibm_scc_posture_scopes.go